src/main/java/jpa/domain/Member.java

```diff
@@ -1,40 +1,37 @@
-package jpa.step1.domain;
+package jpa.domain;
 
-import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
 
+import javax.persistence.CascadeType;
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
 import javax.persistence.Table;
 
-import org.hibernate.annotations.CreationTimestamp;
-
 @Table(name = "member")
 @Entity
-public class Member {
+public class Member extends BaseEntity {
 
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	@Column(name = "id")
 	private Long id;
 
 	@Column(name = "age")
-	private Integer age;
+	private final Integer age;
```

Comment:
```suggestion
	private final int age;
```

특별한 이유가 없다면 primitive type을 추천합니다

Reply:
여기서는 생성자 변수로 값을 입력 받고 있기 때문에 final 처리를 해서 not null한 필드가 되었는데요.
만약 nullable한 필드라면 래퍼 클래스를 사용하는게 맞겠죠?

Reply:
말씀하신 것처럼 nullable 허용한다면 래퍼를 쓰는게 맞습니다.